backend/gce_test.go

```diff
@@ -176,6 +176,7 @@ func gceTestSetup(t *testing.T, cfg *config.ProviderConfig, resp *gceTestRespons
 		cfg = config.ProviderConfigFromMap(map[string]string{
 			"ACCOUNT_JSON": "{}",
 			"PROJECT_ID":   "project_id",
+			"IMAGE_ALIAS_": "default",
```

Comment:
It's totes strange, and specific to the tests, and means that language is `""` somewhere...